### PR TITLE
Add documentation for the OPENSSL_gmtime functions

### PR DESCRIPTION
**Pull request by tmshort**
##### Checklist

- [X] documentation is added or updated
- [ ] tests are added or updated

### CODE REVIEW

File: doc/man3/OPENSSL_gmtime.pod [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+It is an error to call OPENSSL_gmtime() with B<result> equal to NULL. The
+contents of the time_t given by B<timer> are stored into the B<result>. Calling
+with B<timer> equal to NULL means use the current time.
+
+OPENSSL_gmtime_adj() converts B<tm> into a days and seconds value, adds the
+offsets, then converts back into a B<struct tm> specified by B<tm>. Leap seconds
+are not considered.
+
+OPENSSL_gmtime_diff() calculates the difference between the two B<struct tm>
+structures B<from> and B<to>. The difference in days is placed into B<pday>,
```
Comment (paulidale) [inline doc/man3/OPENSSL_gmtime.pod:38]: B<*pday> perhaps?
Likewise B<*psec>.

Not sure either way.

**Review by paulidale — APPROVED** [review]
One suggestion which is optional.

File: doc/man3/OPENSSL_gmtime.pod [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+OPENSSL_gmtime() returns the UTC time specified by B<timer> into the provided
+B<result> argument.
+
+OPENSSL_gmtime_adj() adds the offsets in B<offset_day> and B<offset_sec> to B<tm>.
+
+OPENSSL_gmtime_diff() calculates the difference between B<from> and B<to>.
+
+=head1 NOTES
+
+It is an error to call OPENSSL_gmtime() with B<result> equal to NULL. The
```
Comment (t8m) [inline doc/man3/OPENSSL_gmtime.pod:29]: Please use `I<>` when referencing the parameters instead of bold everywhere in the manpage.

**Review by t8m — CHANGES_REQUESTED** [review]

**Review by t8m — APPROVED** [review]

**Review by paulidale — APPROVED** [review]

**Comment by hlandau** [comment]
Merged to master and 3.0. Thank you.

**Pull request closed without merging**